libs/reentrancy/README.md

```diff
@@ -0,0 +1,19 @@
+# Overview
+
+The Reentrancy library provides an API to check for and disallow reentrancy on a contract.
+
+More information can be found in the [specification](./SPECIFICATION.md).
+
+## Known Issues
+
+## Using the Library
+
+### Using the Reentrancy Guard
+
+Once imported, using the Reentrancy Library can be done by calling one of the two functions. For
+more information, see the [specification](./SPECIFICATION.md).
+
+- `is_reentrant() -> bool`
+- `reentrancy_guard()`
```

Comment:
How do you feel about changing this to an example like so?

```rust
fn my_function() {
     reentrancy_guard();
     // My code here
}
```

Reply:
Good idea, done!